packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -86,15 +83,15 @@ const PlansIntervalToggle: React.FunctionComponent< ToggleHostProps > = ( {
 					<span className="plans-interval-toggle__label">
 						{ __( 'Annually', __i18n_text_domain__ ) }
 					</span>
-					{ intervalType === 'MONTHLY' && (
+					{ intervalType === 'MONTHLY' && typeof maxMonhtlyDiscountPercentage !== undefined && (
```

Comment:
```suggestion
					{ intervalType === 'MONTHLY' && maxMonhtlyDiscountPercentage && (
```

Reply:
`maxMonhtlyDiscountPercentage` could technically be `0` and fail this check  — I think that checking explicitly for it to be undefined is better in this instance

Reply:
Good point but would we want to show "save up to 0%"?

Reply:
That's also a good point. I'll leave it up to you — I'm good with both options. I'll just add a comment in any case

Reply:
It's a nit anyways. I'd go for `&& maxMonhtlyDiscountPercentage` because it covers two cases (0 and undefined).

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/48963/commits/c71139b0fc0d0f20421f1dbca88a1a327acaa187